src/sql/workbench/services/query/common/queryRunner.ts

```diff
@@ -419,6 +419,7 @@ export default class QueryRunner extends Disposable {
 
 	/**
 	 * Get more data rows from the current resultSets from the service layer
+	 * @deprecated getQueryRowsPaged should be used instead as it is much more performant
```

Comment:
I don't think this should be made deprecated just yet - until the new API is ready to be used.

Reply:
Not sure what you mean - the "new" API is the one we switched back to months ago. We should probably investigate the usage of this across the rest of ADS - but that's outside of the scope of this PR which is specifically only targeting the Notebook scenario.

Reply:
And to be a bit clearer - I don't see any problems with the new API itself. Rather, it's with how the Notebook code specifically is using it that's problematic. But fixing that is a much bigger task - so in the interest of getting a fix out quickly (especially for possible hotfix) I'm doing this now and then will be following up with @chlafreniere on further investigation/fixes as needed.

Reply:
@alanrenmsft How confident are you on the non-Notebook usage of this new API? I'll leave it up to you and @cheenamalhotra to discuss whether you want to roll this all back...